tiledb/sm/array_schema/array_schema.cc

```diff
@@ -1405,7 +1445,7 @@ ArraySchema ArraySchema::deserialize(
       capacity,
       attributes,
       dimension_labels,
-      {},
+      std::vector<shared_ptr<const Enumeration>>(),
```

Comment:
I assume this was because of a compiler error when `{}` couldn't be used to initialize a tdb::pmr::vector instance?

Reply:
correct!

Reply:
Thanks!